README.md

```diff
@@ -56,29 +56,28 @@ A [IIIF 2.1 Image API](https://iiif.io/api/image/2.1/) compliant server written
 1. Make sure you have the [SAM CLI](https://aws.amazon.com/serverless/sam/) and [AWS CLI](https://aws.amazon.com/cli/) installed.
 2. Make sure the AWS CLI is [properly configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) with credentials that have sufficient access to manage IAM, S3, Lambda, and (optionally) CloudFront resources.
 3. Clone this repository.
-4. Open a shell in the working copy's `sam/standalone` or `sam/cloudfront` directory, depending on which flavor you would like to install.
+4. Copy .env.example to .env. Update the various values within.
```

Comment:
these readme updates are about the build scripts I added for my connivence. We can pull them out if you'd rather have people work with SAM directly.